ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -62,17 +64,20 @@ def pose_cb(self, msg):
 
     def waypoints_cb(self, msg):
         self.wps = msg.waypoints
+        self.wps_base = copy.deepcopy(self.wps)
         for w in self.wps: w.twist.twist.linear.x = 0
 
     def cruise_trajectory(self, next_wp):
         curr_vel = float(self.wps[next_wp-1].twist.twist.linear.x)
         last_wp = next_wp + LOOKAHEAD_WPS
         wps = self.wps[next_wp : last_wp]
+        wps_base = self.wps_base[next_wp : last_wp]
         # inc_vel = (MAX_VEL - curr_vel)/float(last_wp - next_wp)
         inc_vel = 1.0
         for i, w in enumerate(wps):
             w.twist.twist.linear.x = curr_vel + inc_vel * (i+1)
             if w.twist.twist.linear.x > MAX_VEL: w.twist.twist.linear.x = MAX_VEL
```

Comment:
This check needs to be removed (and the global variable). Else, higher velocities in configuration will not work

Reply:
yep, if we can really drive car at max velocity provided by udacity (specified in ros/src/waypoint_loader/launch/waypoint_loader.launch). I did verify it before push commit and my car ran out of track at speed > ~30 MPH, can be slow hardware, but will be nice if you can verify it on your hardware after I push this change.